pkg/rekor/verifier.go

```diff
@@ -100,6 +105,23 @@ func VerifyConsistencyCheckInputs(interval *time.Duration, logInfoFile *string,
 	return nil
 }
 
+func verifyCertificateTransparencyConsistency(logInfoFile string, logClient *ctclient.LogClient, signedTreeHead *ct.SignedTreeHead) error {
```

Comment:
We'll also need signature verification for the previous STH and current STH, like with https://github.com/sigstore/rekor-monitor/blob/d7458ab1ec52524bf11057d4c2e38bf07b6a5a5c/pkg/rekor/verifier.go#L56. Checkpoints are signed, which provides integrity for the checkpoint.

We can hardcode the public key for now from https://github.com/sigstore/root-signing/blob/main/targets/ctfe_2022.pub, but we should add a TODO to fetch it from our TUF repository like what we need to do for rekor.